plugins/module_utils/user.py

```diff
@@ -477,7 +486,37 @@ def pick(x):
         else:
             return x
 
+    mysql8_all_privileges = [
```

Comment:
```suggestion
    mysql8_all_privileges = [
```
if we can name these groups with meaningful names, wouldn't a dict serve better here? especially later when invoking the sublists? Also the second list look sorted

Reply:
@Andersson007 honestly I don't know. I barely understand the output of `show grants for root@localhost` on MySQL 8 (I only use MariaDB and MySQL 5.7.). Why 2 lines ? How to call the keys in the dictionary? "Basic" and "Advanced"?

Reply:
@laurent-indermuehle call them as you like, though i would make them fully lowercased